cc-raisely-components/components/custom-form/custom-form.js

```diff
@@ -575,6 +576,15 @@
 			return builtSteps;
 		}
 
+		// updates number of guests to 1 if user accidentally enters 0
+		updateNumberOfGuests = async () => {
+
+			if (this.state.values[0].guests === "0") {
+				this.setState({values: [{...this.state.values[0], guests: "1"}]})
```

Comment:
Nick have you tested this?
`setState` is an asynchronous function, so the change to state is not guaranteed to have updated by the time the function returns.
I'd suggest you instead just make a copy of this.state.values, return that from this function, and pass that into `save`

Reply:
Hey Chris, yeap I tested it and it works, but I think your method is more efficient so I should do that instead! 
I can remove the updateNumberOfGuests function as well